apertium_apy/apy.py

```diff
@@ -178,11 +185,17 @@ def apply_config(args, parser, apy_section):
 
 
 def parse_args(cli_args=sys.argv[1:]):
+    if(found is False):
+        location = langNamesDefault
+    else:
+        path = pkg_resources.resource_filename('apertium_apy', langNamesDefault)
+        location = path[0: len(path) - 25] + langNamesDefault
```

Comment:
Please use path parsing methods available in the `os` module.

Reply:
Shouldn't I use pkg_resources?